ros_ws/src/simulation/racer_description/urdf/racer.xacro

```diff
@@ -3,6 +3,10 @@
 <robot name="myrobot" 
   xmlns:xacro="http://www.ros.org/wiki/xacro">
 
+  <xacro:arg name="use_gpu" default="true"/>
+  <xacro:arg name="visualize_lidar" default="false"/>
+  <xacro:arg name="laser_sample_count" default="1080"/>
```

Comment:
Since these are only used in `racer_plugins.gazebo`, the default values should be set there.

Reply:
Done.